modules/runner/src/integrationTest/java/org/apache/ignite/internal/table/ItInternalTableTest.java

```diff
@@ -384,78 +359,190 @@ public void testRoScanAllExplicitPopulatingData() throws InterruptedException {
         roScanAll(false);
     }
 
-    private void roScanAll(boolean implicit) throws InterruptedException {
-        IgniteImpl node = node();
+    @Test
+    public void getAllOrderTest() throws Exception {
+        var keyRows = populateEvenKeysAndPrepareEntriesToLookup(true);
 
         InternalTable internalTable = ((TableImpl) table).internalTable();
+        var schemaDescriptor = ((TableImpl) table).schemaView().schema();
 
-        KeyValueView<Tuple, Tuple> keyValueView = table.keyValueView();
+        List<BinaryRow> res = internalTable.getAll(keyRows, null).get();
```

Comment:
It is not correct to call `CompletableFuture#get` without a timeout, please use `org.apache.ignite.internal.testframework.matchers.CompletableFutureMatcher#willCompleteSuccessfully`.